spec/selinux_state_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe 'selinux_state_test::default' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new.converge(described_recipe)
+  end
+
+  it 'enforcing selinux' do
+    expect(chef_run).to enforcing_selinux_state('enforcing')
+  end
```

Comment:
Not sure what you mean. There is a line break right after the line you commented on?